### PR TITLE
Updating experiment name in postRegister

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request updates a name we updated in #975 

### How should this be reviewed?

👀 

### Any background context you want to provide?

Spotted by @mendelB 🕵 

### Relevant tickets

References [Pivotal #170666913](https://www.pivotaltracker.com/story/show/170666913).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

**Comment by mendelB** [comment]
I wonder if it would be overkill or helpful, to have some constant definition someplace for this sort of thing 🤔

**Comment by DFurnes** [comment]
> I wonder if it would be overkill or helpful, to have some constant definition someplace for this sort of thing 🤔 (maybe in `config/`?)

I really like that idea! 👍

File: app/Http/Controllers/Web/AuthController.php [hunk 1]
```diff
@@ -190,7 +190,7 @@ public function getRegister()
      */
     public function postRegister(Request $request)
     {
-        convert('social-auth-first');
+        convert('social-auth-position');
```
[the remark this answers is not in the document]
Comment (lindsayrmaher) [reply app/Http/Controllers/Web/AuthController.php]: oh great catch Katie, I totally did not think about that! We want to convert them within the `handleCallback` right? And only if they're successful? Or do we only care about whether they're redirected not actually registered in order to count it as a conversion?

[the remark this answers is not in the document]
Comment (lindsayrmaher) [reply app/Http/Controllers/Web/AuthController.php]: @katiecrane I updated to include the convert on `Google` and `FB`, lemme know if this feels like we've covered all our bases!

File: app/Http/Controllers/Web/FacebookController.php [hunk 2]
```diff
@@ -96,6 +96,8 @@ public function handleProviderCallback()
         } else {
             $fields['email'] = $email;
 
+            convert('social-auth-position');
```
Comment (katiecrane) [inline app/Http/Controllers/Web/FacebookController.php]: I think we might want to move this to be below the creation of `$northstarUser` so we only convert them when they are successfully registered

Comment (katiecrane) [reply app/Http/Controllers/Web/FacebookController.php]: And in Google too!

File: app/Http/Controllers/Web/AuthController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (katiecrane) [reply app/Http/Controllers/Web/AuthController.php]: I think we also want to move this conversion down to after the user is successfully registered, does that make more sense?

File: app/Http/Controllers/Web/FacebookController.php (hunk 2 above)
Comment (lindsayrmaher) [reply app/Http/Controllers/Web/FacebookController.php]: yeah yeah that makes sense, for all three to be post successful creation of NS user.

**Pull request merged**